client/curriculum/html/AudioAndVideo.tsx

```diff
@@ -14,13 +14,20 @@ import Highlight from '~/components/Highlight/Highlight';
 import { DemoPreview } from '~/components/demo';
```

Comment:
Same as in the `BoxModel.tsx`.

Let's keep the file as is, and only modify where we have to replace with `<FormattedText>`.